extensions/amp-story/1.0/pagination-buttons.js

```diff
@@ -330,7 +335,7 @@ export class PaginationButtons {
     if (pageIndex === totalPages - 1) {
       this.ampStory_.hasBookend().then((hasBookend) => {
         if (!hasBookend) {
-          this.forwardButton_.updateState(ForwardButtonStates.REPLAY);
+          this.forwardButton_.updateState(ForwardButtonStates.NEXT_STORY);
```

Comment:
I think we only want to do this when inside of the player/viewer. We don't want to apply this logic to all stories.